app/2.0/docs/devguide/dom-template.md

```diff
@@ -14,18 +14,18 @@ The DOM template is also processed to enable data binding and declarative event
 
 Polymer provides three basic ways to specify a DOM template:
 
--   `<dom-module>` element. This allows you to specify the template entirely in markup,
-    which is most efficient for an element defined in an HTML import.
--   Provide a string template. This works well for elements defined entirely in JavaScript
-    (for example, in an ES6 module).
--   Retrieve or generate your own template element. This allows you to define how the template is
-    retrieved or constructed, and can be used to modify a superclass template.
+-   [Specify a template using the `<dom-module>` element](#dommodule). This allows you to specify the
+    template entirely in markup, which is most efficient for an element defined in an HTML
+    import.
+-   [Assign a string template to an HTMLTemplateElement object, and return the 
```

Comment:
I like the new links.

I think the description of this one focuses too much on the implementation. The gist here is that you can define a `template` getter that returns the template. 

Except that @justinfagnani raised a concern about the wording of that. Um, so,...

"Add a template property on the constructor that returns the template?"

"Provide a template by defining a template property on the constructor"?